src/test/resources/custom_fields.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I think it is helpful to include some reference to the redmine version into test file names.
e.g. "issue_created_by_redmine_2.2.1.json"